recipes/singlem/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# install python libraries
+python -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+
+# copy main singlem python script
+chmod +x bin/singlem
```

Comment:
If this actually necessary? Upstream lists `bin/singlem` as a script that should be installed by `pip install`. If the missing executable bit is the problem, can you please report this upstream?

Reply:
Good point!